### PR TITLE
[MO] setup stripe

### PR DESCRIPTION
**Pull request by GuillaumeRenouvin**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/mo-setup-stripe/react-native/setup/setup-stripe-dev-standard.mo.md">setup-stripe-dev-standard.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: react-native/setup/setup-stripe-dev-standard.mo.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+### [MO] Setup Stripe in app payment form with tispi-stripe (~2 hours)
+
+#### Owner [Guillaume Renouvin](https://github.com/GuillaumeRenouvin)
+
+#### Prerequisites
+- Have a Stripe account and its publishable/secret key duet
```
Comment (tychota) [inline react-native/setup/setup-stripe-dev-standard.mo.md]: Where to found it ? which url ?

File: react-native/setup/setup-stripe-dev-standard.mo.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+#### However your client should be aware of
+- [tipsi-stripe](https://github.com/tipsi/tipsi-stripe) only supports english;
+- [tipsi-stripe](https://github.com/tipsi/tipsi-stripe) has not been audited by Stripe yet. The PCI DSS (Payment Card Industry Data Security Standard) regulation committee recently forced Stripe to stop accepting credit card data from direct calls to their API. Therefore, they enforced the usage of "Stripe-controlled" libraries like Elements and their own SDKs.  
+**It means, Stripe could actually ask you to stop using tipsi in order to fit new PCI DSS requirements anytime.**  
+So far, tipsi-stripe uses direct calls to Stripe SDKs and therefore, we did not receive any warning from them, still you should keep it in mind.
+
+However, even if Stripe asked you to stop using tipsi, the time loss would be light as it only takes around 2 hours to install the lib. Setting up browser payment would take more than a day (Ledon, Neo-nomade, ...) and offers a poorer UX.
```
Comment (tychota) [inline react-native/setup/setup-stripe-dev-standard.mo.md]: No client name, please.

File: react-native/setup/setup-stripe-dev-standard.mo.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+
+#### Steps (~1 hour)
+- Install tipsi-stripe package following his [installation documentation](https://github.com/tipsi/tipsi-stripe#installation)
```
Comment (tychota) [inline react-native/setup/setup-stripe-dev-standard.mo.md]: How to check it works after install:
- break point in ios in the code
- break point in android in the code
- console.log(NativeModules) where NativeModules is a react native import

**Review by tychota — APPROVED** [review]
Sinon, au top !

**Pull request merged**